flutter/ios/Classes/SwiftSentryFlutterPlugin.swift

```diff
@@ -4,247 +4,278 @@ import UIKit
 
 public class SwiftSentryFlutterPlugin: NSObject, FlutterPlugin {
 
-  private var sentryOptions: Options?
+    private var sentryOptions: Options?
 
-  public static func register(with registrar: FlutterPluginRegistrar) {
-    let channel = FlutterMethodChannel(name: "sentry_flutter", binaryMessenger: registrar.messenger())
-    let instance = SwiftSentryFlutterPlugin()
-    registrar.addMethodCallDelegate(instance, channel: channel)
-  }
+    // The Cocoa SDK is init. after the notification didBecomeActiveNotification is registered.
+    // We need to be able to receive this notification and start a session when the SDK is fully operational.
+    private var startSession = false
```

Comment:
what about a timestamp?

Reply:
that makes sense, if the exposed method or notification receives it, we could use as session start

Reply:
we've decided to not do it for now because it'd bring extra complexity, right now there was no method on Cocoa that would take a timestamp, so every class would need to be changed/new overload.